R/key-helpers.R

```diff
@@ -27,8 +62,9 @@ check_key <- function(.data, ...) {
   data_q <- enquo(.data)
   .data <- eval_tidy(data_q)
   args <- exprs(...)
```

Comment:
We could do `names(args) <- set_names(paste0("...", seq_along(args)))`, then we're sure that the `n` column is always returned. Can you please isolate these changes to a separate PR?